packages/components/src/components/tick-picker/tick-picker.tsx

```diff
@@ -14,7 +14,7 @@ type TTickPicker = {
     min_value: number;
     max_value: number;
     onSubmit: (props: { target: { value: number; name: string } }) => void;
-    submit_label: React.ReactElement;
+    submit_label: React.ReactElement | string;
```

Comment:
probably should have been React.ReactNode?

Reply:
@henry-deriv no no, React.ReactNode will also include null and false, etc. which is not suitable here unfortunately :(